docs/main_text.md

```diff
@@ -362,20 +362,27 @@ Experiment metadata (e.g., experiment description and title) is delivered in the
 
 # Species compendia
 
-refine.bio is currently in beta.
-Once refine.bio reaches production, we will periodically release compendia comprised of all the samples from a species that we were able to process.
+We periodically release compendia comprised of all the samples from a species that we were able to process.
 We refer to these as **species compendia**.
-We'll process these compendia in a manner that is different from the options that are available via the web user interface.
-We describe our intended processing pipeline below.
-Instead of selecting only genes available in all samples, we take the union of all genes, filling in any missing values with `NA` (e.g., perform a full outer join as illustrated below).
+We process these compendia in a manner that is different from the options that are available via the web user interface.
+
+Instead of selecting only genes available in all samples, we take the union of all genes, filling in any missing values with `NA`. 
+This is a "full outer join" as illustrated below. 
 
 ![outer join](https://user-images.githubusercontent.com/15315514/44534241-4dde9100-a6c5-11e8-8a9c-aa147e294e81.png)
 
+We perform an outer join each time samples are combined in the process of building species compendia.
+
+![docs-species-compendia](https://user-images.githubusercontent.com/15315514/48498088-72995f00-e803-11e8-8832-3a9024748431.png)
 
-We drop any genes that have missing values in greater than 30% of samples.
-We impute the remaining missing values with KNN impute.
+Samples from each technology—microarray and RNA-seq—are combined separately.
+In RNA-seq samples, we filter out genes with low total counts and then `log2(x + 1)` the data.
+We join samples from both technologies, drop genes that have missing values in greater than 30% of samples, and drop samples that have missing values in greater than 50% of genes.
```

Comment:
Just to check, you drop samples first, then genes?

Reply:
No, we drop genes (https://github.com/AlexsLemonade/refinebio/blob/a27543a83ebe6946fe87ed082a64458ad7d57b12/workers/data_refinery_workers/processors/create_compendia.py#L137) and then samples (https://github.com/AlexsLemonade/refinebio/blob/a27543a83ebe6946fe87ed082a64458ad7d57b12/workers/data_refinery_workers/processors/create_compendia.py#L145). I'll include your suggested change below.